pnpm-lock.yaml

```diff
@@ -13821,6 +14019,7 @@ packages:
       command-exists: 1.2.9
       console.table: 0.10.0
       detect-libc: 2.0.2
+      zx: 7.2.3
```

Comment:
this is one of the places new deps are coming from. `rust: nightly` has a new dependency